torjail

```diff
@@ -252,6 +252,12 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
+TORBIN=$(which tor 2>/dev/null1)
```

Comment:
Oh, after making the pull request I found that you had prepared TOREXEC= variable in the header. Well, I think TORBIN aligns better with other ...BIN= variable you used ;)